source/MaterialXView/Material.cpp

```diff
@@ -481,9 +481,9 @@ void Material::bindLights(mx::LightHandlerPtr lightHandler, mx::GLTextureHandler
     // Bind environment light uniforms and images.
     if (specularEnvironmentMethod == mx::SPECULAR_ENVIRONMENT_FIS)
     {
-        if (_glShader->uniform("u_envSamples", false) != -1)
+        if (_glShader->uniform("u_envRadianceSamples", false) != -1)
```

Comment:
Should this come from the identifier map instead of being hard-coded?

Reply:
Good point, I will fix this so both tokens and identifier names are accessible.